packages/waku/src/lib/hono/runner.ts

```diff
@@ -28,6 +28,12 @@ export const runner = (options: MiddlewareOptions): MiddlewareHandler => {
         .map(async (middleware) => (await middleware).default(options)),
     ),
   );
+
+  (globalThis as any).__WAKU_HACK_IMPORT__ = async (id: string) => {
```

Comment:
`__WAKU_HACK_IMPORT__` is unused, right?

btw, I don't like this hack. there must be a better way.

Reply:
yes, React will read buildConfig and use __WAKU_HACK_IMPORT__ to load the react server action

Reply:
Alright, `__WAKU_HACK_IMPORT__` should only be required for client. Let me see if I can do something.